platform/ui/src/utils/ModalProvider.js

```diff
@@ -0,0 +1,91 @@
+import React, { useState, createContext, useContext } from 'react';
+import Modal from 'react-bootstrap-modal';
+
+const ModalContext = createContext(null);
+const { Provider, Consumer } = ModalContext;
+
+export const useModalContext = () => useContext(ModalContext);
+
+const ModalProvider = ({ children }) => {
+  const DEFAULT_OPTIONS = {
+    component: null,
+    backdrop: false,
+    keyboard: false,
+    show: true,
+    large: true,
+    closeButton: true,
+    title: null,
+    customClassName: null,
```

Comment:
Can we get inline comments for these to better indicate what they do?

Reply:
Sure! done! thx.